### ISSUE TITLE
load the RVL_CDIP for LayoutLMv3ForSequenceClassification

### ISSUE THREAD

**Issue by wytszhuzhu** [microsoft/unilm#996]
when i follow this [https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv2/RVL-CDIP/Fine_tuning_LayoutLMv2ForSequenceClassification_on_RVL_CDIP.ipynb](url) load the RVL_CDIP for LayoutLMv3ForSequenceClassification. I find the load time is very slow(For hours) and have some warning.  Change the batch_size still don't work. Can you tell me how to deal with it. Thanks.

from transformers import LayoutLMv3FeatureExtractor, LayoutLMv3Tokenizer, LayoutLMv3Processor
from datasets import load_dataset
from datasets import Features, Sequence, ClassLabel, Value, Array2D, Array3D

dataset = load_dataset("rvl_cdip", split="train")
feature_extractor = LayoutLMv3FeatureExtractor()
tokenizer = LayoutLMv3Tokenizer.from_pretrained("microsoft/layoutlmv3-base")
processor = LayoutLMv3Processor(feature_extractor, tokenizer)

labels = ['letter', 'form', 'email', 'handwritten', 'advertisement', 'scientific report', 'scientific publication', 'specification', 'file folder', 'news article', 'budget', 'invoice', 'presentation', 'questionnaire', 'resume', 'memo']
id2label = {v: k for v, k in enumerate(labels)}
label2id = {k: v for v, k in enumerate(labels)}

features = Features({
    'pixel_values': Array3D(dtype="float32", shape=(3, 224, 224)),
    'input_ids': Sequence(feature=Value(dtype='int64')),
    'attention_mask': Sequence(Value(dtype='int64')),
    'bbox': Array2D(dtype="int64", shape=(512, 4)),
    'labels': ClassLabel(num_classes=len(labels), names=labels),
})

def preprocess_data(examples):
      images = [path.convert("RGB") for path in examples['image']]
    encoded_inputs = processor(images, padding="max_length", truncation=True)
    encoded_inputs["labels"] = examples['label']
    return encoded_inputs

encoded_dataset = dataset.map(preprocess_data, remove_columns=dataset.column_names, features=features,
                              batched=True, batch_size=2)

![image](https://user-images.githubusercontent.com/49281157/218989683-4b6df671-0458-44e9-93bf-6d56a435a966.png)
